eslint.config.js

```
@@ -1,40 +1,75 @@
-        project: './tsconfig.json'
-      }
+        ecmaFeatures: {
+          jsx: true,
+        },
+      },
+      globals: customGlobals,
     },
     plugins: {
-      '@typescript-eslint': tsPlugin,
-      'jest': jestPlugin
+      'react-native': reactNativePlugin
```

ESLint

[error] Insert `,`

(prettier/prettier)